scripts/gha/install_prereqs_desktop.py

```diff
@@ -32,11 +32,20 @@
 import utils
 
 def main():
+  # Install protobuf on linux/mac if its not installed already
+  if not utils.is_command_installed('protoc'):
```

Comment:
vcpkg's installation of protobuf has protoc in the final build directory. 
But to use it, you have to use the vcpkg cmake toolchain. I am guessing this is for some other script being used outside cmake?

Reply:
I don't run protoc myself but I was getting build errors without this - maybe just because vcpkg wasn't supporting x86 linux yet, so that build was failing. Should I leave this in (until that's fixed) or remove it?

Reply:
Oh interesting. Yeah, we shouldn't need to explicitly have protobuf installed. Let it be there for now. I have a pending PR and will test your workflow out in my PR (after building x86 correctly).